tests/accelerators/test_accelerator_connector.py

```diff
@@ -457,10 +459,10 @@ def test_accelerator_gpu():
     assert trainer._device_type == "gpu"
     assert isinstance(trainer.accelerator, GPUAccelerator)
 
-    with pytest.raises(
-        MisconfigurationException, match="You passed `accelerator='gpu'`, but you didn't pass `gpus` to `Trainer`"
-    ):
-        trainer = Trainer(accelerator="gpu")
+    # with pytest.raises(
+    #     MisconfigurationException, match="You passed `accelerator='gpu'`, but you didn't pass `gpus` to `Trainer`"
+    # ):
```

Comment:
Reviewers: this error doesn't make sense? Auto device here?

Reply:
I personally advocate for setting device=auto here.

There's a discussion in https://github.com/PyTorchLightning/pytorch-lightning/issues/10606 if you want to chime in.

Reply:
Will track by #10606